jekyll/__glossary/owner.md

```diff
@@ -0,0 +1,5 @@
+---
+term: Owner
+---
+
+The owner of the team GitHub or Bitbucket account (org).
```

Comment:
The (org) at the end confuses me. Perhaps change to:

"The owner of the team GitHub account or Bitbucket org."